src/features/signUp/signUpInput/Email.tsx

```diff
@@ -29,11 +41,13 @@ export default function Email() {
         placeholder="이메일 (예: example@gmail.com)"
         value={email}
         onChange={handleEmailChange}
-        isValid={isEmailValid}
+        isValid={isEmailValidLocal}
         name="email"
         type="text"
       />
-      {!isEmailValid && <ErrorMessage>{emailError}</ErrorMessage>}
+      {(!isEmailValidLocal || emailError) && (
+        <ErrorMessage>{emailError || localEmailError}</ErrorMessage>
+      )}{' '}
```

Comment:
```suggestion
      {emailError && <ErrorMessage>{emailError}</ErrorMessage>}
```

하나의 상태로만 관리해도 될 것 같아요